ansible/roles/common/files/authorised_keys/pvaneynde.pub

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHjErAEeV7XmajochqXCGqu3YaLKsfNnCksX/WcySRZ6 pevaneyn@PEVANEYN-M-50LF
```

Comment:
You don't want to split roles like this, keys should be in `ansible/roles/users/files/foo.pub`